migrations.py

```diff
@@ -25,8 +25,8 @@ async def m001_initial(db):
         f"""
         CREATE TABLE fossa.fossa_payment (
             id TEXT NOT NULL PRIMARY KEY,
-            deviceid TEXT NOT NULL,
```

Comment:
has this version not been release yet?
shouldn't be a new migration that renames the columns?

Reply:
it has not been released, people are still using lnurldevice for fossa